cmd/app/command/device-list.go

```diff
@@ -18,11 +18,28 @@ package command
 
 import (
 	goflag "flag"
+	"fmt"
+	"html/template"
+	"os"
 
 	"github.com/openebs/node-disk-manager/cmd/controller"
 	"github.com/spf13/cobra"
 )
 
+// defaultDeviceList template use to print list of device
+const defaultDeviceList = `
+{{- if .Items}}
+	{{- printf "%-45s" "NAME"}}{{printf "%-20s" "MODEL"}}{{printf "%-20s" "VENDOR"}}
```

Comment:
Let us put each header item in a new line

Reply:
done.